### PR TITLE
Excludes vars and var_edited from VV list

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by kevinz000** [comment]
Isn't this for admins to view if something was tampered with too?

**Comment by Cyberboss** [comment]
if `var_edited` is true you get the big red banner up top

**Comment by lzimann** [comment]
Is it a problem having those showing?

**Comment by Cyberboss** [comment]
No, but they will always be null ops so it's pointless

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -139,6 +139,8 @@
 	var/list/names = list()
 	if (!islist)
 		for (var/V in D.vars)
+			if(V == "vars" || V == "var_edited")
+				continue
```
Comment (MrStonedOne) [inline code/datums/datumvars.dm]: Please do not add snowflake to VV code, i worked hard to remove it.

add this to vv_get_var or what ever its called.

**Comment by MrStonedOne** [comment]
I'm mildly against this change because knowing what vars are on a thing can be important too, even if you can't read them or they will always be null.

**Pull request closed without merging**